src/mesh/Mesh.hpp

```diff
@@ -47,12 +47,21 @@ class Mesh : public PropertyContainer, private boost::noncopyable
   typedef utils::ptr_vector<Quad>                QuadContainer;
   typedef std::vector<PtrData>                   DataContainer;
   typedef utils::ptr_vector<PropertyContainer>   PropertyContainerContainer;
+  /// a vector of pairs that represents the physical range of vertices owned by a rank. 
   typedef std::vector<std::pair<double, double>> BoundingBox;
+  /// set of boundingboxes owned by each rank gathered in the master rank.
+  /// A map : each rank -> boundingbox around it's mesh partition.
   typedef std::map<int,BoundingBox>              BoundingBoxMap;
+  /// a map from rank -> a vector of ranks : shows each rank is connected
+  /// to which ranks of the other participant. 
+  typedef std::map<int, std::vector<int>>        FeedbackMap; 
```

Comment:
Is it really necessary to have this data structure as member of the mesh? Couldn't it simply be a local variable during the feedback process. We should only attach data structures to the mesh if they have to be carried from one package to another, here, normally, from partition to m2n.